src/test/java/de/refactoringbot/refactorings/RemoveParameterTest.java

```diff
@@ -91,9 +88,9 @@ private void testRemoveParameter(int lineNumberOfMethodWithParameterToBeRemoved,
 				removeParameterTestClass.getLineNumberOfDummyMethod(0, 0, 0), cuOriginalFile);
 		MethodDeclaration originalCallerMethod = RefactoringHelper
 				.getMethodByLineNumberOfMethodName(removeParameterTestClass.getLineNumberOfCaller(), cuOriginalFile);
-		assertNotNull(originalMethod);
-		assertNotNull(originalDummyMethod);
-		assertNotNull(originalCallerMethod);
+		assertThat(originalMethod).isNotNull();
```

Comment:
when having a lot of assertions in one testcase it is better to use `SoftAssertions` and call all the assertions on that object. The test will not fail on the first assertion but is able to execute all of them so you get the feedback of every failed assertion and not only the first one. Don't forget to call `softAssertions.assertAll()` at the end. There should be already one example where I used it in my tests.